libmscore/score.h

```diff
@@ -1310,6 +1311,7 @@ class MasterScore : public Score
     TimeSigMap * _sigmap;
     TempoMap* _tempomap;
     RepeatList* _repeatList;
+    RepeatList* _repeatList2;
```

Comment:
Please tell why it was necessary to add `_repeatList2`.
If it is impossible otherwise, a detailed commentary is needed for what `_repeatList2` is used and for what `_repeatList`, and what is the difference between them.
It is also better to give them clear names.